caom2-search-server/examples/default/build.gradle

```diff
@@ -28,7 +28,9 @@ dependencies {
 
 ext {
     // Change this value to test other locations, such as Production or RC.
+//    intTest_default_web_app_url = 'https://rc.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/en/search/'
     intTest_default_web_app_url = 'https://rc.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/en/search/'
+    intTest_web_app_url = 'http://localhost:4444/search'
```

Comment:
Please change this back as it won't exist in Travis.  It will override the `default` value at runtime.

Reply:
reverted.